talk/basicconcepts/functions.tex

```diff
@@ -248,7 +248,7 @@
             // Step 2: muons
             for (....) {
               if (...) {
-                data.delete(...);
+                data.erase(...);
```

Comment:
`delete` was syntax highlighted here, but I guess a member function was meant here and not `operator delete`.